tests/configfiles/DockerfileLintBearTest.py

```diff
@@ -23,7 +24,40 @@
 apt-get -y -qq upgrade
 """
 
+good_file_nodejs = """
+FROM ubuntu
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+
+RUN apt-get -y -qq update
+RUN apt-get -y -qq upgrade
+RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+EXPOSE 5432
+CMD ["/usr/bin/node", "/var/www/app.js"]
+"""
+
+bad_file_nodejs = """
+FROM ubuntu
+
+apt-get -y -qq update
+apt-get -y -qq upgrade
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+"""
+
+rule_file = os.path.join(os.path.dirname(__file__),
```

Comment:
A default rule file is used if no file is already specified for dockerfile_lint. I think you are using that default file only. Maybe try using a custom file and see if the tests pass

Reply:
@anctartica Yes, a default rule_file is used when no rule_file is provided while running this bear. I've included the same default file so that the results are comparable to the previous test i.e. the test which was already written for this bear. Although, I did test it with some other rules. Let me know if it's still required to change it. I'll add another test. ;)

Reply:
Yes, I would try to add a custom rule file and use that for the tests to make sure everything is working as intended 👍

Reply:
@anctartica made the changes ;)